web/admin/src/editors.js

```diff
@@ -65,34 +66,59 @@ export class MultiLineEditor extends PureComponent {
   }
 
   render() {
-    const {title, placeholder, validationMessage} = this.props
+    const {title, validationMessage} = this.props
 
     return (
-      <label className="text-editor">
-        <div className="text-editor__title">{title}</div>
-        { this.isValid() ? null : <div className="text-editor__error">{validationMessage}</div> }
+      <div className="multiline-editor__box">
+        <div className="editorBox__header">
+          {this.renderButton("text", "Text", 1)}
```

Comment:
It seems like the specific concern of HTML content is bleeding into this generic multi-line editor, which shouldn't be concerned with what the content is (other than that it is multi-line text)